instagram/src/CommentSection.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      input: '',
+      comments: this.props.comments
```

Comment:
No need for this as you already have access to 'props' because they're being pulled in the constructor.

Reply:
in fact, it might be better you use componentDidMount to set the comments onto the state of this component.